src/utils.ts

```diff
@@ -67,35 +82,63 @@ function printElement({ props }) {
   )}`;
 }
 
-function display(value) {
+function display(value: any) {
```

Comment:
```suggestion
function display(value: unknown) {
```

Reply:
Fixed, added eslint rule for the future that checks explicit any.